src/widgets/ActivityDetails/model/hooks/useAnswers.ts

```diff
@@ -155,18 +155,20 @@ export const useAnswer = (props: Props) => {
       return answer;
     },
     [
-      encryptPayload,
-      generateUserPrivateKey,
-      getGroupProgress,
-      props.activityId,
-      props.applet.activityFlows,
       props.applet.encryption,
+      props.applet.activityFlows,
       props.applet.id,
       props.applet.version,
+      props.flowId,
+      props.activityId,
       props.eventId,
       props.eventsRawData,
-      props.flowId,
+      encryptPayload,
+      getGroupProgress,
+      featureFlags.enableMultiInformant,
+      generateUserPrivateKey,
       getMultiInformantState,
+      isInMultiInformantFlow,
```

Comment:
Same here. Can we revert the changes to the dependency array?

Reply:
Done